test/e2e/shared/cluster.go

```diff
@@ -91,6 +91,7 @@ func setupBootstrapCluster(config *clusterctl.E2EConfig, scheme *runtime.Scheme,
 			Name:               config.ManagementClusterName,
 			RequiresDockerSock: config.HasDockerProvider(),
 			Images:             config.Images,
+			KubernetesVersion:  config.GetVariable(KubernetesVersion),
```

Comment:
I noticed that the kind cluster was using a different version from the workload clusters. Probably it makes more sense to use the same version, so I put it here.

Reply:
Reverted. There is no kind node image with this tag.